packages/kbn-test/src/functional_tests/cli/run_tests_cli.js

```diff
@@ -0,0 +1,63 @@
+import getopts from 'getopts';
+import { runTests } from '../../';
+import {
+  FUNCTIONAL_CONFIG_PATH,
+  API_CONFIG_PATH,
+} from '../../functional_tests/lib';
+
+const defaultConfigPaths = [FUNCTIONAL_CONFIG_PATH, API_CONFIG_PATH];
+
+/**
+ * Run servers and tests for each config
+ * Only cares about --config option. Other options
+ * are passed directly to functional_test_runner, such as
+ * --bail, --verbose, etc.
+ * In the future, this method will accept --tests-only,
+ * --servers-only.
+ * @param {configPaths}     array of paths to configs
+ */
+export function runTestsCli() {
+  const {
+    configs,
+    help,
+  } = processArgv();
+
+  if (help) return displayHelp();
+
+  runTests(configs);
+}
+
+
+function processArgv() {
+  const options = getopts(process.argv.slice(2)) || {};
+  const configs = [];
+
+  try {
```

Comment:
__issue:__ hmm, it seems we can just use `if (options.config !== undefined) {` instead of `try/catch`.

Reply:
not just undefined, but also if it's not an array (a typeof string). i could go either way. either explicitly call out what conditions it can't be like you suggest, or what i do currently-- say no matter what the failure, let's use defaults.

Reply:
i think you're right here. let me make sure.

Reply:
Simplified it much more